{
  "context": "Final Generate Failure after all retries",
  "datetime": "2025-04-02T11:56:44.739901",
  "stack_trace": "NoneType: None\n",
  "exception": "ERROR: Failed to get a valid response after 2 retries."
}